core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitor.scala

```diff
@@ -197,6 +213,24 @@ object V1WorkspaceMigrationMonitor
         .update((tmpBucketName.value.some, Timestamp.valueOf(LocalDateTime.now).some))
     ))
   }
+
+  final def claimAndConfigureNewGoogleProject(migrationAttempt: V1WorkspaceMigrationAttempt,
+                                              workspaceService: WorkspaceService,
+                                              workspace: Workspace,
+                                              billingProject: RawlsBillingProject
+                                             ): Future[(GoogleProjectId, GoogleProjectNumber, WriteAction[Unit])] =
+    for {
+      (googleProjectId, googleProjectNumber) <- workspaceService.setupGoogleProject(
```

Comment:
There's a few more things in the ticket that aren't done by `setupGoogleProject`. If we don't want to do those things for this ticket, then we need to move them to an existing ticket or create a new one to track that work.

* Add the Google project id and the Google project number to the DB for this workspace record [in the `WORKSPACE` table] (I expect this just needs to be moved to a different ticket since we're saving the project id and project number in our migration history table)
* Add workspace's Google project to a service perimeter if workspace's Terra billing project is in a service perimeter
* Create Sam `google-project` resource with existing `workspace` resource as the parent

Reply:
Yeah sure. We won't be updating the workspace record until the very last moment. I thought we'd update the record and sam once the migration was done. Service perimeters are covered by a separate ticket

Reply:
The security perimeter ticket seems to be more focused on figuring out when the perimeter update has actually finished propagating. The service perimeter update action can still be added now IMO. If we don't want to add it now, then we should note in the security perimeter ticket that we also need to do the update.

Is there a ticket for updating the record and Sam when the migration is done? I don't want those steps to be lost in the shuffle.